src/main/java/com/tragent/pressing/service/implementation/PaymentServiceImpl.java

```diff
@@ -16,7 +16,7 @@
 import com.tragent.pressing.service.PaymentService;
 
 @Service
-@Secured({"ROLE_MANAGEMENT", "ROLE_SALES_AGENT"})
+@Secured({"ROLE_MANAGEMENT", "ROLE_SALES_AGENT", "ROLE_ADMINISTRATION"})
```

Comment:
I did not look at it well, Whats the difference between ROLE Administrator and Management?